models/tts/valle/valle_trainer.py

```diff
@@ -209,17 +209,17 @@ def _train_epoch(self):
 
         return epoch_sum_loss, epoch_losses
                
-    def _get_state_dict(self):
-        state_dict = {
-            "model": self.model.state_dict(),
-            "optimizer": self.optimizer.state_dict(),
-            "scheduler": self.scheduler.state_dict(),
-            "step": self.step,
-            "epoch": self.epoch,
-            "batch_size": self.cfg.train.batch_size,
-            "trian_stage": self.args.train_stage
-        }        
-        return state_dict
+    # def _get_state_dict(self):
```

Comment:
why is this impacted?

Reply:
It's the testing code before. And unuseful now. I have deleted it.